src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +33,26 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
+              DisplayName="Framework tools"
+              Description="Compiler to use when building the project"
+              Category="General">
+    <EnumValue Name="true"
+             DisplayName=".NET Framework" />
+    <EnumValue Name="false"
+             DisplayName=".NET SDK" />
+  </EnumProperty>
+
+  <EnumProperty Name="FSharpPrefer64BitTools"
```

Comment:
This property feels like it should be a boolean (check box) to me. That would match what we have for ["Prefer 32-bit" property](https://github.com/dotnet/project-system/blob/e9f3bbd77e6e1e8d151e06580d469a1150745973/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml#L100-L123).

![image](https://user-images.githubusercontent.com/350947/131767770-dab92502-80bc-493f-af81-1934eadd2c5c.png)

Reply:
I changed our (C#) Prefer 32-bit property because we should be consistent with boolean values always being represented as checkboxes. I definitely recommend the same here.